### PR TITLE
Pixel: add button styles

### LINKED ISSUE #6457: Pixel: Styles: Buttons
Update button styles.

Button double borders (box-shadow) can be configured in theme.json. Related GB feature: https://github.com/WordPress/gutenberg/pull/41972

Design: Cm51iCQSPu8Ar1qufDTeOQ-fi

image

### PR DESCRIPTION
**Pull request by jffng**
#### Changes proposed in this Pull Request:

This PR adds button styles to Pixel's theme.json.

Default | Hover | Click
--- | --- | ---
Screen Shot 2022-09-01 at 2 38 56 PM | Screen Shot 2022-09-01 at 2 39 03 PM | Screen Shot 2022-09-01 at 2 39 09 PM

#### Related issue(s):

Closes #6457

### CODE REVIEW

File: pixel/theme.json [hunk 1]
```diff
@@ -343,26 +339,47 @@
...
+                    "text": "var(--wp--preset--color--primary)"
                 },
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--medium)",
-                    "fontWeight": "normal",
-                    "lineHeight": "2",
+                    "fontSize": "var(--wp--preset--font-size--small)",
                     "textDecoration": "none"
                 },
+                "shadow": "var(--wp--custom--shadow)",
```
Comment (madhusudhand) [inline pixel/theme.json]:
```suggestion
                "shadow": "var(--wp--custom--shadow) !important",
```

File: pixel/theme.json [hunk 2]
```diff
@@ -343,26 +339,47 @@
...
+                },
+                ":active": {
+                    "color": {
+                        "background": "var(--wp--preset--color--background)",
+                        "text": "var(--wp--preset--color--primary)"
+                    },
+                    "border": {
+                        "color": "var(--wp--preset--color--primary)"
+                    },
+                    "shadow": "2px 2px"
```
Comment (madhusudhand) [inline pixel/theme.json]:
```suggestion
                    "shadow": "2px 2px !important"
```

**Review by madhusudhand — COMMENTED** [review]
Changes are woking good, except for the following two scenarios.

* no shadow for File button on hover
image

* no shadow for comment button
image

Both are because there exists a `box-shadow: none;` style set by gutenberg with higher specificity.
I will create an issue to lower the specificity.

For now we could add `!important` to shadow to gain higher specificity.

**Comment by jffng** [comment]
Thanks @madhusudhand your suggestion fixed both those cases, so going to merge this one:

**Button Block, File Block**
Screen Shot 2022-09-02 at 10 52 49 AM

**Post Comment**
Screen Shot 2022-09-02 at 10 52 51 AM

**Pull request merged**